widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidgetView.kt

```diff
@@ -55,7 +57,9 @@ internal class EntryWidgetView(
     }
 
     override fun showItems(items: List<EntryWidgetContract.ItemType>) {
-        viewAdapter.items = items
+        Handler(Looper.getMainLooper()).post {
```

Comment:
What is the purpose of upgrading items property on main thread?

Reply:
Otherwise, UI is not updated dynamically.